coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -125,11 +127,16 @@ def apply(self,
             if tuple(current_file) != tuple(new_file):
                 print_beautified_diff(difflib.unified_diff(current_file,
                                                            new_file,
-                                                           fromfile=filename,
+                                                           fromfile=(
```

Comment:
I am not sure about this. Won't it act weird? I never seen or wrote any code that changes value of the variable that is obtained after iterating... I'll try this